plugins/module_utils/lxd.py

```diff
@@ -57,10 +57,20 @@ def __init__(self, url, key_file=None, cert_file=None, debug=False):
         self.debug = debug
         self.logs = []
         if url.startswith('https:'):
-            self.cert_file = cert_file
-            self.key_file = key_file
+            try:
+                with open(cert_file, 'r') as fh:
+                    self.cert_file = cert_file
+            except FileNotFoundError:
+                raise LXDClientException('The certificate is not valid.')
```

Comment:
How about:
```suggestion
                raise LXDClientException('The certificate "{0}" does not exist.'.format(cert_file))
```
(With a similar change below.)